src/renderer/views/Search/Search.js

```diff
@@ -38,11 +38,15 @@ export default defineComponent({
     },
 
     backendPreference: function () {
-      return this.$store.getters.getBackendPreference
+      let preference = this.$store.getters.getBackendPreference
```

Comment:
**suggestion (non-blocking):** Would be nice to have this as a helper function, both to reduce the code duplication, & to better clarify the intent of when it's included. Also lets us only have to update the logic in one place if we ever need to update it.

Reply:
This code will be removed in future PRs. This code is here because I didn't add support for searching with Piped as this PR would be much larger and harder to review (and it will be removed in other places once it gets implemented in Piped)